### DISCUSSION TITLE
How to fetch manual time input?

### DISCUSSION THREAD

**Discussion by JCarloZ24** [danniehansen/activity-timer#139] [Q&A]
Hello there, I was wondering how to get the manual time input from the Activity timer using JavaScript to gsheets?
@
![image](https://github.com/danniehansen/activity-timer/assets/166006451/2ee7e9b2-93a1-4a09-a52c-e6cca11e975d)

**Comment by danniehansen** [comment]
Hi @JCarloZ24,

You would need run some server-side code with a language of your choice and hosting of your choice that can run scheduled when you need to transfer the records.

More information: https://github.com/danniehansen/activity-timer/tree/master?tab=readme-ov-file#docs

The data is recorded in the pluginData of the powerup and accessible through the RESTApi :+1:
